### PR TITLE
Migration/pedalmarking

### PR DESCRIPTION
**Pull request by rvilarl**
PedalMarking

@0xfe here I had to resolve a bug mixing pedal type and element style.

### CODE REVIEW

File: src/pedalmarking.ts [hunk 1]
```diff
@@ -9,86 +9,116 @@
 // custom text for the release/depress pedal markings.
 
 import { Vex } from './vex';
-import { Element } from './element';
+import { Element, ElementStyle } from './element';
 import { Glyph } from './glyph';
+import { FontInfo, RenderContext } from './types/common';
+import { StaveNote } from './stavenote';
+
+export interface PedalMarkingRenderOptions {
```
Comment (0xfe) [inline src/pedalmarking.ts:17]: This can be inline.

File: src/pedalmarking.ts [hunk 2]
```diff
@@ -9,86 +9,116 @@
...
+  color: string;
+  bracket_height: number;
+  text_margin_right: number;
+  bracket_line_width: number;
+}
+
+export interface PedalMarkingGlyph {
```
Comment (0xfe) [inline src/pedalmarking.ts:24]: This can be inline too.

**Review by 0xfe — COMMENTED** [review]
Some minor comments.

File: src/pedalmarking.ts (hunk 2 above)
Comment (rvilarl) [reply src/pedalmarking.ts:24]: done

**Comment by rvilarl** [comment]
Ready to merge!

**Comment by 0xfe** [comment]
Looks good, and thanks for fixing the bug.

**Pull request merged**